### ISSUE TITLE
Allow specifying variables when queueing builds

### ISSUE THREAD

**Issue by lzybkr** [Microsoft/vsts-cli#105]
We have a mandatory parameter in our build that is specified in the Web UI.

If I try to queue such a build from `vsts`, I get an error:

```
PS> vsts build queue --definition-name Build_Test --verbose -o json
'NoneType' object has no attribute 'message'
```

**Comment by tedchamb** [comment]
@lzybkr Thanks for reporting.  We should definitely add support for this.

**Comment by lzybkr** [comment]
I did debug some more and found the error has nothing to do with my request, I'll open another issue for that.

**Comment by bergmeister** [comment]
+1.
It would also be nice to get prompted if the build has mandatory variables with no default.

**Comment by tedchamb** [comment]
addressed in #109
